DependentExtensions/Lobby2/PGSQL/Lobby2Message_PGSQL.cpp

```diff
@@ -2696,7 +2695,7 @@ bool SLNet::Ranking_GetTotalScore_PGSQL::ServerDBImpl( Lobby2ServerCommand *comm
 		return true;
 	}
 
-	int64_t count;
+	long long count;

```

Comment:
I think `int64_t` and any other `[u]int<size>_t` types are better than plain ones. Maybe static_ or reinterpret_cast on the line below will be better?

Reply:
Yes, it generally better to know exact size of variable regardless of target platform. However, PQGetValueFromBinary do not have a corresponding overload for int64_t, but for long long.